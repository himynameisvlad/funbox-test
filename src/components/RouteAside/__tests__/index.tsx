import { shallow } from 'enzyme';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { RouteAsideComponent } from '../index';

jest.mock('ui', () => ({
  FormInput: () => <div />,
}));
jest.mock('../RouteList', () => ({
  RouteList: () => <div />,
}));

describe('RouteAside - aside component', () => {
  let wrapper;
  const onSubmit = () => null;
  const RouteAside = <RouteAsideComponent onSubmit={onSubmit} />;

  beforeEach(() => {
    wrapper = shallow(RouteAside);
  });

  it('renders correctly', () => {
    const tree = renderer.create(RouteAside).toJSON();

    expect(tree).toMatchSnapshot();
    expect(wrapper.find('FormInput').length).toBe(1);
    expect(wrapper.find('RouteList').length).toBe(1);
  });
});
