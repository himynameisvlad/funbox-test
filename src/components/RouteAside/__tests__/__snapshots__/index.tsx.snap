// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteAside - aside component renders correctly 1`] = `
Array [
  <div />,
  <div
    className="fb-routes-aside__list"
  >
    <div />
  </div>,
]
`;
